dotcom-rendering/src/experiments/tests/sign-in-gate-main-variant.ts

```diff
@@ -7,7 +7,7 @@ export const signInGateMainVariant: ABTest = {
 	author: 'Mahesh Makani',
 	description:
 		'Show sign in gate to 100% of users on 3rd article view of simple article templates, and show a further 5 times after the first dismissal, with higher priority over banners and epic. Main/Variant Audience.',
-	audience: 0.9,
```

Comment:
I think we need to wait to release this PR as this will turn off the main gate because no on will be in the bucket?  Or we leave at .9 and do the switch ON via the frontend Admin Switches?

Can you add a link to the correspondedn Frontend PR for the AB switches? I think the Jan test switch can be deleted and replaced.

Reply:
Sure, yeah we can use the switches to control this.

Reply:
Added PR for frontend in the description